packages/libs/eda/src/lib/map/analysis/MapHeader.tsx

```diff
@@ -88,60 +74,7 @@ export function MapHeader({
           onAnalysisNameEdit={onAnalysisNameEdit}
         />
       </div>
-      {outputEntity && (
-        <div className={`${mapHeader('__SampleCounter')}`}>
-          <p>{makeEntityDisplayName(outputEntity, true)}</p>
-          <LeftBracket
-            styles={{
-              // Bring closer the content of the righthand side of
-              // the bracket.
-              marginLeft: 10,
-            }}
-          />
-          <table>
-            <thead>
-              <tr>{/* <th colSpan={2}>{entityDisplayName}</th> */}</tr>
-            </thead>
-            <tbody>
-              <tr
-                title={`There are ${format(
-                  totalEntityCount
-                )} ${makeEntityDisplayName(
-                  outputEntity,
-                  totalEntityCount > 1
-                )} in the dataset.`}
-              >
-                <td>All</td>
-                <td>{format(totalEntityCount)}</td>
-              </tr>
-              <tr
-                title={`After filtering, there are ${format(
-                  totalEntityInSubsetCount
-                )} ${makeEntityDisplayName(
-                  outputEntity,
-                  totalEntityInSubsetCount > 1
-                )} in the subset.`}
-              >
-                <td>Filtered</td>
-                <td>{format(totalEntityInSubsetCount)}</td>
-              </tr>
-              <tr
-                title={`${format(visibleEntityCount)} ${makeEntityDisplayName(
-                  outputEntity,
-                  visibleEntityCount > 1
-                )} are in the current viewport${
-                  overlayActive
-                    ? ', and have data for the painted variable'
-                    : ''
-                }.`}
-              >
-                <td>View</td>
-                <td>{format(visibleEntityCount)}</td>
-              </tr>
-            </tbody>
-          </table>
-        </div>
-      )}
+      {mapTypeDetails}
```

Comment:
Is it OK to expect the `mapTypeDetails` content/component to position itself to the right of the header, or should we use a container similar to `HangingTabs`?

Reply:
I believe the container uses `justify-content: space-between`, which will keep the content positioned to the right.

Reply:
https://github.com/VEuPathDB/web-monorepo/blob/a4a752828cd9ce49129bb2357df88644dc62dceb/packages/libs/eda/src/lib/map/analysis/MapHeader.scss#L5